packages/enr/src/creator.ts

```diff
@@ -0,0 +1,36 @@
+import { PeerId } from "@libp2p/interface-peer-id";
+import type { ENRKey, ENRValue } from "@waku/interfaces";
+import { utf8ToBytes } from "@waku/utils";
+
+import { compressPublicKey } from "./crypto.js";
+import { ENR } from "./enr.js";
+import { getPublicKeyFromPeerId } from "./peer_id.js";
+
+export class EnrCreator {
```

Comment:
Instead of having a class with static function, should I just use free functions?

Reply:
I think both are good